libs/api/src/main/java/de/cyface/api/model/TrackBucket.java

```diff
@@ -1,112 +0,0 @@
-/*
- * Copyright 2021 Cyface GmbH
- *
- * This file is part of the Cyface Data Collector.
- *
- * The Cyface Data Collector is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * The Cyface Data Collector is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with the Cyface Data Collector. If not, see <http://www.gnu.org/licenses/>.
- */
-package de.cyface.api.model;
-
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-
-import de.cyface.model.MetaData;
-import de.cyface.model.Track;
-import io.vertx.core.json.JsonObject;
-
-/**
- * The mongo database "bucket" which contains a slice of a track.
- *
- * @author Armin Schnabel
- * @since 6.0.0
- */
-public class TrackBucket {
```

Comment:
This model moved to `serialization/libs/model`, see other PR.

Reply:
`serialization` 2.1.0 now contains this model, the Upgrade to 2.1.0 was done in the previous commit (due to rebasing this moved)